

push int16(32 ;)
pu int(32))
exit

Does the program stops properly because of a syntax error?

